test/e2e/pageLevelProgress.cy.js

```diff
@@ -0,0 +1,41 @@
+describe('Page Level Progress', function () {
+  beforeEach(function () {
+    cy.getData()
+    cy.visit('/')
+  });
+  
+  it('should display the page level progress bars correctly on the menu items', function () {
+    const isPageLevelProgressEnabled = this.data.course._pageLevelProgress._isEnabled
```

Comment:
We might need to guard here for courses that don't have the extension added
```suggestion
    const isPageLevelProgressEnabled = this.data.course._pageLevelProgress?._isEnabled
```